code/game/objects/items/sharpener.dm

```diff
@@ -1,57 +1,77 @@
+/**
+* # Whetstone
+*
+* Items used for sharpening stuff
+*
+* Whetstones can be used to increase an item's force, throw_force and wound_bonus and it's change it's sharpness to SHARP_EDGED. Whetstones do not work with energy weapons. Two-handed weapons will only get the throw_force bonus. A whetstone can only be used once.
+*
+*/
 /obj/item/sharpener
 	name = "whetstone"
 	icon = 'icons/obj/kitchen.dmi'
 	icon_state = "sharpener"
 	desc = "A block that makes things sharp."
-	force = 5
-	var/used = 0
+	force = 5 //This is how hard the whetstone itself hits stuff
```

Comment:
you don't need to doc this, it should be documented where it's first defined as a variable, rather then here.